assets/src/routes/routes.js

```diff
@@ -3,23 +3,23 @@ import { Grade } from '@material-ui/icons'
 const routes = (courseId, activeViews) => {
   const allViews = [
     {
-      path: `/${courseId}/grades`,
+      path: `/courses/${courseId}/grades`,
       title: 'Grade Distribution',
       icon: Grade,
       description: 'See where your grade sits within the course grade distribution.',
       image: '/static/images/grade_distribution_icon.png',
       viewCode: 'gd'
     },
     {
-      path: `/${courseId}/assignment`,
+      path: `/courses/${courseId}/assignment`,
```

Comment:
This is changed to `/courses/${courseId}/assignments`

Reply:
fixed here as well as in the Course commponent